### ISSUE TITLE
CVE-2023-29337 in azure-functions-host

### ISSUE THREAD

**Issue by atpoirie** [Azure/azure-functions-docker#922]
Using image python:4-python3.10-appservice, started seeing CVE-2023-29337, marked high, being identified in reference to nuget.protocol and nuget.common in the file /azure-functions-host/Microsoft.Azure.WebJobs.Script.WebHost.deps.json

Not sure if this belongs on the azure functions docker side, azure functions host side, or both.

Vulnerability is new, but assuming other commercial entities will start reporting the same when their deployments become blocked.

**Comment by abl-jakev** [comment]
We are also running into this being flagged by our security scanner.

**Comment by BaileyFirman** [comment]
We are also seeing this; I haven't dug too deep but it looks like Debian's package repository doesn't have an updated version of [NuGet](https://www.apt-browse.org/browse/debian/stretch/main/all/nuget/2.8.7+md510+dhx1-1/?query=nuget) yet so it may be blocked by that.

**Comment by vivekjilla** [comment]
Thank you for reporting this and for providing the vulnerability details.

We checked `CVE-2023-29337` against all 17 current supported base images and did not find the reported CVE IDs in any of those image reports.

Please rebuild your application from the latest supported image and let us know if you still see the same finding.

Azure Functions publishes regular base-image security updates, with monthly updates documented publicly:

- [Supported Azure Functions languages and runtime versions](https://learn.microsoft.com/azure/azure-functions/supported-languages)
- [Maintaining Azure Functions custom containers](https://learn.microsoft.com/azure/azure-functions/functions-how-to-custom-container)
- [Azure Functions images in MCR](https://mcr.microsoft.com/en-us/catalog?search=azure-functions)

We are closing this issue based on the current image scan. If the same finding remains on a current supported digest, please open a new issue or request reopening and copy the finding details in this format:

```text
Image:
Digest:
Scanner:
Scanner version:
Vulnerability database date:
CVE:
Package:
Target/path:
Installed version:
Fixed version:
Severity:
```

You can also attach the raw scanner output if convenient.
